src/Microsoft.Identity.Web/AppServicesAuth/AppServicesAuthenticationInformation.cs

```diff
@@ -10,173 +10,173 @@
-            }
-        }
+		/// <summary>
+		/// Client secret of the App Services Auth web site.
+		/// </summary>
+		internal static string? ClientSecret
+		{
+			get
+			{
+				return Environment.GetEnvironmentVariable(AppServicesAuthClientSecretEnvironmentVariable);
+			}
+		}
 
-        /// <summary>
-        /// Issuer of the App Services Auth web site.
-        /// </summary>
-        internal static string? Issuer
-        {
-            get
-            {
-                return Environment.GetEnvironmentVariable(AppServicesAuthOpenIdIssuerEnvironmentVariable);
-            }
-        }
+		/// <summary>
+		/// Issuer of the App Services Auth web site.
+		/// </summary>
+		internal static string? Issuer
+		{
+			get
+			{
+				return Environment.GetEnvironmentVariable(AppServicesAuthOpenIdIssuerEnvironmentVariable);
+			}
+		}
 
 #if DEBUG
-        /// <summary>
-        /// Get headers from environment to help debugging App Services authentication.
-        /// </summary>
-        internal static string? SimulateGetttingHeaderFromDebugEnvironmentVariable(string header)
-        {
-            string? headerPlusValue = Environment.GetEnvironmentVariable(AppServicesAuthDebugHeadersEnvironmentVariable)
-                ?.Split(';')
-                ?.FirstOrDefault(h => h.StartsWith(header));
-            return headerPlusValue?.Substring(header.Length + 1);
-        }
+		/// <summary>
+		/// Get headers from environment to help debugging App Services authentication.
+		/// </summary>
+		internal static string? SimulateGetttingHeaderFromDebugEnvironmentVariable(string header)
+		{
+			string? headerPlusValue = Environment.GetEnvironmentVariable(AppServicesAuthDebugHeadersEnvironmentVariable)
+				?.Split(';')
+				?.FirstOrDefault(h => h.StartsWith(header));
+			return headerPlusValue?.Substring(header.Length + 1);
+		}
 #endif
 
-        /// <summary>
-        /// Get the ID token from the headers sent by App services authentication.
-        /// </summary>
-        /// <param name="headers">Headers.</param>
-        /// <returns>The ID Token.</returns>
-        internal static string? GetIdToken(IDictionary<string, StringValues> headers)
-        {
-            if (headers is null)
-            {
-                throw new ArgumentNullException(nameof(headers));
-            }
+		/// <summary>
+		/// Get the ID token from the headers sent by App services authentication.
+		/// </summary>
+		/// <param name="headers">Headers.</param>
+		/// <returns>The ID Token.</returns>
+		internal static string? GetIdToken(IDictionary<string, StringValues> headers)
+		{
+			if (headers is null)
+			{
+				throw new ArgumentNullException(nameof(headers));
+			}
 
-            string? idToken = null;
-            if (headers.ContainsKey(AppServicesAuthIdTokenHeader))
-            {
-                idToken = headers[AppServicesAuthIdTokenHeader];
-            }
+			string? idToken = null;
+			if (headers.ContainsKey(AppServicesAuthIdTokenHeader))
```

Comment:
TryGetValue results in a single dictionary lookup rather than two lookups when using ContainsKey with subsequent access.
```suggestion
			if (headers.TryGetValue(AppServicesAuthIdTokenHeader, out var idToken))
```